pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,64 @@
 # Image classification
-
-Models that are able to classify images.
+## Description of the models
+### Performance results pivot table
 
 | Model Name | Complexity (GFLOPs) | Size (Mp) | Top-1 on ImageNet (%) | Links |
 | ---        | ---                 | ---       | ---                   | ---                   |
 | MobilenetV2 1.0 | 0.66           | 3.50      | 73.0                  | [model template](./mobilenet_v2_w1/template.yaml)
 
-## Datasets
-
-The model was initially trained on [ImageNet](http://image-net.org/challenges/LSVRC/2012/), but can be trained from scratch or fine tuned to classify arbitrary images.
-
+| Model Name | Complexity (GFLOPs) | Size (Mp) | Latency (ms) | FPS | AVG mAP (%) | AVG Top-1 (%) | AVG Top-5 (%) | Links |
+| ---        | ---        | ---        | ---        | ---        | ---        | ---        | ---        | ---        |
+| EfficientNet b0 | 0.76 | 4.14 | 9.7 | 447.51 | 92.267 | 90.156 | 98.283 | [model template](./efficient_b0/template.yaml) |
+| MobilenetV3 large x1.0 | 0.44 | 4.33 | 5.58 | 792.22 | 83.46	| 87.06 | 96.42 | [model template](./mobilenet_v3_large_1/template.yaml) |
+| MobilenetV3 large x0.75 | 0.308 | 2.84 | 4.48 | 985.4 | 87.59 |	87.59	| 97.17 | [model template](./mobilenet_v3_large_075/template.yaml) |
+| MobilenetV3 small x1.0 | 0.112 | 1.56 | 2.79 | 1599.39 | 82.84 | 84.47 | 95.88 | [model template](./mobilenet_v3_small/template.yaml) |
+
+All of the above metrics were obtained on eleven different datasets, on which an extensive number of research had been made.
```

Comment:
has?

Reply:
2 @kprokofi 
Sorry, Kirill, but could you point what do you mean?
What mean `AVG mAP(%)` in this case?  The `mAP` metric is calculated for each of the eleven dataset, and then they were averaged?
But AFAIK the datasets have different number of classes, so you cannot use one model for all of them?

Reply:
You're right, the mAP metric as well as TOP-1 and TOP-5 are calculated for each dataset and then averaged. We need some representative average metric to sum up the model's performance result. I agree, maybe it's not very good to average metrics, but I haven't come up with another idea. Also, it seems useful in practice to recognize the best config for the "average" classification task in this way

Reply:
Kirill, I'm sorry, but for me it is totally unclear. 
When I wrote my assumption that you averaged metrics among several datasets, I thought that it is not true -- I thought that you will write "Obviously not, we measured on ImageNet", or something like that 🙂 

And I think that this approach should be described explicitly.
At least, since indeed the models for different datasets are **different**, since they have different last layer.
So, the models should be **finetuned** on these datasets.
In my opinion you really should insert description how you finetuned the models on the datasets.
As an example, I really do not understand, if you freeze all the weights except the last layer during the finetuning and how many epochs did you do.

In my opinion:
* If you *did not* freeze all the weights except the last layer you *should* write it, since indeed the models may be finetuned by all their weights 
  I understand that the difference will be small, but it should be described explicitly, these details should not be hidden
* If you *did* freeze all the weights except the last layer you *should* write it, since this is an additional operation, that should be described explicitly too

Reply:
thanks for the detailed comment. I added descriptions to README for this.